tuareg.el

```diff
@@ -72,6 +72,7 @@
 
 ;;; Code:
 
+
```

Comment:
Just a missing call to my-just-one-line afterwards. Will fix this with next commit.